### ISSUE TITLE
Instantiator downloaded filename should be {FamilyName}-{InstanceName}

### ISSUE THREAD

**Issue by graphicore** [Lorp/samsa#36]
Just noticed while watching the presentation.

**Comment by Lorp** [comment]
Good idea, though there are issues to solve when InstanceName is not a Named Instance or derived from STAT. I also wonder if it’s important to maintain a relationship with the original filename, so maybe simply append the style name to that. Probably the auto-naming needs multiple modes :) BTW in samsa-cli.js you can override instance filename with the `--output` option.

**Comment by graphicore** [comment]
> I also wonder if it’s important to maintain a relationship with the original filename, so maybe simply append the style name to that.

Yes, sounds like a good start.
